README.md

```diff
@@ -118,3 +101,200 @@ Remote debugging using :2000
 Black Magic Debug App also provides for Flashing, reading and verification of a binary file,
 by default starting at lowest flash address. The `-t` argument displays information about the
 connected target. Use `-h`/`--help` to get a list of supported options.
+
+#### OS specific remarks
+
+On *BSD and macOS, using `/dev/tty.usbmodemXXX` should work but unresolved discussions indicate a
+hanging `open()` call on the second invocation. If that happens, try with `/dev/cu.usbmodemXXX`.
```

Comment:
The default recommendation for at least macOS should be the `/dev/cu.usbmodemXXX`. The reason are the caches which are present in the tty version that cause hangups and all kinds of trouble. People should by default use `cu` devices.

Reply:
That should be taken care of now - please give the new wording a check and resolve this if you're happy.

Reply:
Great! Thanks :)